src/objects/Location.php

```diff
@@ -126,6 +131,36 @@ public function getCoords()
         return ($this->Lat != 0 && $this->Lng != 0) ? 'true' : 'false';
     }
 
+    /**
+     * @return string
+     */
+    public function getCategoryList()
```

Comment:
This could be shortened a bit here:
```php
if($this->Categories()->count()) {
    return implode(', ', $this->Categories()->column('Name'));
}

return '';
```